### ISSUE TITLE
Wrong Shortcut Name After Renaming Assembly

### ISSUE THREAD

**Issue by JRKelso** [Squirrel/Squirrel.Windows#548]
So during initial development we had named our application in the pattern of "App.Name.exe" now that we are close to release are changing the assembly for it to "Diff-App-Name.exe". We've updated the nuspec and everything to include the new exe names instead of the old one as the old one is no longer generated. Before the assembly name change it was automatically generating the desktop shortcut and start menu shortcuts (we weren't explicitly calling to create these shortcuts from code). Now whenever we run a new build with the new exe names and clean fresh installs of the application it creates a desktop shortcut with the old name still. Even calling SquirrelAwareApp with a CreateShortcutForExe or CreateShortcutsForExecutable doesn't generate the appropriate shortcut. Is there some settings that needs to be changed for this?

**Comment by enoumen** [comment]
I have the same issue. Did you ever get a fix for this?

**Comment by JRKelso** [comment]
Thanks to source control I have some evidence as to what I changed to get this to work, though I'm not 100% as to which of these things fixed it, probably is all of them in combination. 

It was just an oversight on my part that I didn't change all the references to the old exe everywhere. So I changed my Assembly Name and thought that was it, however it turned out that when I was packaging my nuspec into a nupkg I was manually setting the Id and the Title to the old name and needed to switch it to the new one, and also had overlooked changing the Assembly Information to the dashed name as well.
